src/components/v5/common/ActionSidebar/hooks/useGetActionData.ts

```diff
@@ -50,7 +51,7 @@ export const useGetActionData = (transactionId: string | undefined) => {
         return {
           [ACTION_TYPE_FIELD_NAME]: ACTION.SIMPLE_PAYMENT,
           amount: {
-            amount,
+            amount: moveDecimal(amount, -(token?.decimals || 0)),
```

Comment:
This needs to use `getTokenDecimalsWithFallback`

```suggestion
            amount: moveDecimal(amount, -(getTokenDecimalsWithFallback(token.decimals))),
```

Reply:
@rdig Fixed in all places